docker-compose.yml

```diff
@@ -8,19 +8,13 @@ services:
       - "8545:8545"
       - "8546:8546"
     volumes:
-      - type: bind
-        source: ./volumes/geth
-        target: /var/lib/geth/data
+      - geth:/var/lib/geth/data
   postgres:
     image: "postgres:12"
     logging:
       driver: none 
-    ports:
```

Comment:
did you remove the port on purpose?

Reply:
yes, it is not necessary to do a port binding with the host machine in order for the l2 to be able to reach the psql database. I removed it to avoid a docker error if a dev user has another psql instance running on his pc or server, which is a common case scenario.